Documentation/Books/Manual/Architecture/Replication/README.md

```diff
@@ -38,6 +38,10 @@ factor. The number of _followers_ can be controlled using the
 `replicationFactor` parameter is the total number of copies being
 kept, that is, it is one plus the number of _followers_. 
 
+In addition to the `replicationFactor` we have a `minReplicationFactor`
+that locks down a collection for writing as soon as we have lost too many followers.
+
+
```

Comment:
Remove changes to the file from this PR, covered by https://github.com/arangodb/docs/pull/108/files#diff-ddd01044a87d8eb454d6ab0fba9ddad3